packages/shorebird_cli/lib/src/commands/patch/patch_android_command.dart

```diff
@@ -112,19 +112,11 @@ If this option is not provided, the version number will be determined from the p
     final shorebirdYaml = shorebirdEnv.getShorebirdYaml()!;
     final appId = shorebirdYaml.getAppId(flavor: flavor);
     final app = await codePushClientWrapper.getApp(appId: appId);
-
     final originalFlutterRevision = shorebirdEnv.flutterRevision;
-    final buildProgress = logger.progress('Building patch');
-    try {
-      await buildAppBundle(flavor: flavor, target: target);
-      buildProgress.complete();
-    } on ProcessException catch (error) {
-      buildProgress.fail('Failed to build: ${error.message}');
-      return ExitCode.software.code;
-    }
 
-    final projectRoot = shorebirdEnv.getShorebirdProjectRoot()!;
+    var hasBuiltWithLatestFlutter = false;
```

Comment:
Is this just "haveBuilt"?

Reply:
Or maybe "haveBuiltWithActiveFlutter"?  Which just gets set to "false" whenever we switch?  And then the switch and not-switch cases can share build logic?

Reply:
I see, they can't actually share due to the finally clause in the switching case.  Maybe that all should be moved into a "Build with Fltuter version" which checks if the asked for version is the same as active?